content/spin/api-guides-overview.md

```diff
@@ -14,7 +14,8 @@ The following table shows the status of the interfaces Spin provides to applicat
 | [Redis Trigger](/spin/redis-trigger)                         | Stable   | No  |
 | [Outbound HTTP](/spin/http-outbound)                          | Stable   | Yes   |
 | [Outbound Redis](/spin/redis-outbound)                         | Stable  | Yes   |
-| [Dynamic Configuration](/spin/dynamic-configuration)                         | Experimental | No |
+| [Configuration Variables](/spin/variables)                      | Stable | Yes |
+| [Runtime Configuration](/spin/dynamic-configuration#runtime-configuration)  | Experimental | No |
```

Comment:
The "configuration variables" API covers the "API guide" bit of this page - "runtime configuration" per se doesn't really have an API - so suggest removing this line.